ng2-components/ng2-activiti-tasklist/src/components/activiti-people-search.component.css

```diff
@@ -0,0 +1,18 @@
+:host {
+    width: 100%;
+}
+
+.activiti-label {
+    font-weight: bolder;
+}
+
+.material-icons:hover {
```

Comment:
this has a chance affecting all material icons, can you isolate your case with some additional class, instead of changing `.material-icons` one? for instance `.people-search__icon` and `.people-search__icon:hover` to use with `<i>icon</i>.